docs/src/components/MenuPropControls.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { MenuProps, SelectField, TextField } from '@aws-amplify/ui-react';
+import { DemoBox } from './DemoBox';
+
+export interface MenuPropControlsProps extends MenuProps {
+  setAlign: (value: React.SetStateAction<MenuProps['align']>) => void;
+  setSize: (value: React.SetStateAction<MenuProps['size']>) => void;
+}
+
+interface MenuPropControlsInterface {
+  (props: MenuPropControlsProps): JSX.Element;
+}
+
+export const MenuPropControls: MenuPropControlsInterface = ({
+  setAlign,
+  setSize,
+  align,
+  size,
+}) => {
+  return (
+    <DemoBox primitiveName="Menu">
+      <SelectField
+        label="align"
+        name="align"
+        onChange={(event) => setAlign(event.target.value as MenuProps['align'])}
+      >
+        <option value="start">start</option>
+        <option value="center">center</option>
+        <option value="end">end</option>
+      </SelectField>
+
+      <SelectField
+        label="size"
+        name="size"
+        placeholder="default"
+        onChange={(event) => setSize(event.target.value as MenuProps['size'])}
+      >
+        <option value="small">small</option>
```

Comment:
Should we add a default option here so the user can get back to the default size if after they change it to small or large?

Reply:
It's added by default already via the `placeholder` prop. I actually added it manually and found it to be duplicated. I'm still unsure of whether we should have the `placeholder` API on the SelectField primitive.